### DISCUSSION TITLE
Responsive Typography

### DISCUSSION THREAD

**Discussion by mg-aceik** [mantinedev/mantine#5135] [Q&A]
I was hoping to do something like this on the theme to achieve responsive font sizes on headings:

    headings: {
      sizes: {
        h1: {
          fontSize: {
            base: 'calc(3rem * var(--mantine-scale))',
            lg: 'calc(4rem * var(--mantine-scale))',
          },
          lineHeight: '1.3',
        },

I also need something similar for a default `<p>` tag that is within TypographyStylesProvider.

I had a look through the docs and couldn't find a way. Any help appreciated.

**Comment by rtivital** [comment]
You can apply responsive styles to components with CSS modules – https://mantine.dev/styles/responsive/
To pass styles, use Styles API – https://mantine.dev/styles/styles-api/

**Reply by mg-aceik** [reply]
Thanks. Would I be able to make a feature request for this functionality? Essentially a way to set on the theme's `fontSizes` and `headings` different values for the available breakpoints.

**Reply by mmahalwy** [reply]
@mg-aceik did you end up fixing this?

**Reply by mg-aceik** [reply]
@mmahalwy no, I had to create a custom .scss file just for this. Would be great to have this feature.
